engine/src/main/java/com/redhat/cloud/notifications/templates/Edge.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Edge" to read templates from resources/templates/Edge
+public class Edge implements EmailTemplate {
+
+    @Override
+    public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals("image-creation")) {
+            return Templates.imageCreationTitle();
+        }
+        return Templates.updateDeviceTitle();
+    }
+
+    @Override
+    public TemplateInstance getBody(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals("image-creation")) {
+            return Templates.imageCreationBody();
+        }
+        return Templates.updateDeviceBody();
```

Comment:
For the same reason than my previous comment, this change would be safer:
```suggestion
        if (eventType.equals("image-creation")) {
            return Templates.imageCreationBody();
        } else if (eventType.equals("update-devices") {
            return Templates.updateDeviceBody();
        }
        throw new UnsupportedOperationException(String.format(
                "No email body template for Edge event_type: %s and EmailSubscription: %s found.",
                eventType, type
        ));
```